src/saving/SaveInformation.cs

```diff
@@ -65,4 +71,11 @@ public static SaveInformation CreateInvalid()
             Creator = "Invalid",
         };
     }
+
+    internal static string GetDescriptionForSaveType(SaveType value)
+    {
+        var memInfo = typeof(SaveType).GetMember(value.ToString());
```

Comment:
Is there no better way than to look up this by name?

Reply:
I have not found a better way in the internet

Reply:
I also couldn't find one. Would it be possible to make this into a generic method that would work on any enum? So it would be something like `public static string GetTranslatedDescriptionOfEnumValue<T>(T value): where something`
I don't know if that's possible, just mentioning an idea I had.

Reply:
The thing you requested is called extention method. I'll add one

Reply:
I was thinking of it being made into an extension method as well. It should be limited in scope, though, it should only work on enums.

Reply:
Something like this?: https://github.com/Revolutionary-Games/Thrive/pull/1837/commits/6f2761b3c5eb022eb4d3c4d600b9594caf9bf8cb

Reply:
Yes, though the generic param and returns docs should be single line to conserve a bit of space.